airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcStreamFactory.kt

```diff
@@ -45,8 +44,8 @@ class MysqlJdbcStreamFactory(val base: JdbcAirbyteStreamFactory) : AirbyteStream
         override val type: FieldType,
     ) : MetaField {
         CDC_CURSOR(CdcIntegerMetaFieldType),
-        CDC_UPDATED_AT(CdcOffsetDateTimeMetaFieldType),
-        CDC_DELETED_AT(CdcOffsetDateTimeMetaFieldType),
+        CDC_UPDATED_AT(CdcStringMetaFieldType),
```

Comment:
For my understanding: this is going to put json values in quotes?

Reply:
yes I believe so